### PR TITLE
Query is breaking for security_ip_list. closes#114

### PR DESCRIPTION
**Pull request by bigdatasourav**

### CODE REVIEW

**Comment by bigdatasourav** [comment]
> select security_ip_list,db_instance_id from alicloud_rds_instance where db_instance_id = 'rm-7go155igl16ty92n9'
+------------------+----------------------+
| security_ip_list | db_instance_id       |
+------------------+----------------------+
| 127.0.0.1        | rm-7go155igl16ty92n9 |

**Comment by bigdatasourav** [comment]
> select jsonb_pretty(ip_array_list) as ip_details from alicloud_rds_instance where db_instance_id = 'rm-7gosgb525264xb5j8'
+---------------------------------------------------------------------------------------------------------------------------------+
| ip_details                                                                                                                      |
+---------------------------------------------------------------------------------------------------------------------------------+
| [                                                                                                                               |
|     {                                                                                                                           |
|         "SecurityIPList": "127.0.0.1",                                                                                          |
|         "SecurityIPType": "IPv4",                                                                                               |
|         "WhitelistNetworkType": "MIX",                                                                                          |
|         "DBInstanceIPArrayName": "default",                                                                                     |
|         "DBInstanceIPArrayAttribute": ""                                                                                        |
|     },                                                                                                                          |
|     {                                                                                                                           |
|         "SecurityIPList": "100.104.106.0/24,100.104.160.192/26,100.104.188.0/24,11.59.10.2,11.59.10.3,11.59.6.125,11.59.6.126", |
|         "SecurityIPType": "IPv4",                                                                                               |
|         "WhitelistNetworkType": "MIX",                                                                                          |
|         "DBInstanceIPArrayName": "hdm_security_ips",                                                                            |
|         "DBInstanceIPArrayAttribute": "hidden"                                                                                  |
|     },                                                                                                                          |
|     {                                                                                                                           |
|         "SecurityIPList": "0.0.0.0/0",                                                                                          |
|         "SecurityIPType": "IPv4",                                                                                               |
|         "WhitelistNetworkType": "MIX",                                                                                          |
|         "DBInstanceIPArrayName": "inbound_internet",                                                                            |
|         "DBInstanceIPArrayAttribute": ""                                                                                        |
|     }                                                                                                                           |
| ]                                                                                                                               |

File: alicloud/table_alicloud_rds_instance.go [hunk 1]
```diff
@@ -440,33 +441,12 @@ func tableAlicloudRdsInstance(ctx context.Context) *plugin.Table {
...
-				Name:        "db_instance_ip_array_attribute",
-				Type:        proto.ColumnType_STRING,
-				Hydrate:     getRdsInstanceIPArrayList,
-				Transform:   transform.FromField("DBInstanceIPArrayAttribute"),
-				Description: "The attribute of the IP address whitelist.",
-			},
-			{
-				Name:        "security_ip_type",
-				Type:        proto.ColumnType_STRING,
+				Name:        "ip_array_list",
```
Comment (cbruno10) [inline alicloud/table_alicloud_rds_instance.go:444]: Can you please rename this to `security_ips_src`? In this column, we should not modify any of the fields, even the `SecurityIPList` field, which is a comma delimited list of IPs.

Comment (cbruno10) [reply alicloud/table_alicloud_rds_instance.go:444]: We should also add a column called `security_ips` that's a list of strings, which is composed of the addresses from the `SecurityIPList` fields.

**Review by cbruno10 — CHANGES_REQUESTED** [review]
Please see comments on column changes, thanks!

**Comment by bigdatasourav** [comment]
```
> select security_ips,security_ips_src,db_instance_id from alicloud_rds_instance
+--------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------
| security_ips                                                                                                                   | security_ips_src                                                         
+--------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------
| ["127.0.0.1","100.104.106.0/24,100.104.160.192/26,100.104.188.0/24,11.59.10.2,11.59.10.3,11.59.6.125,11.59.6.126","0.0.0.0/1"] | [{"DBInstanceIPArrayAttribute":"","DBInstanceIPArrayName":"default","Secu
| ["127.0.0.1","100.104.106.0/24,100.104.160.192/26,100.104.188.0/24,11.59.10.2,11.59.10.3,11.59.6.125,11.59.6.126","0.0.0.0/0"] | [{"DBInstanceIPArrayAttribute":"","DBInstanceIPArrayName":"default","Secu
+--------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------
> select security_ips,security_ips_src,db_instance_id from alicloud_rds_instance where db_instance_id='rm-7go155igl16ty92n9'
+--------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------
| security_ips                                                                                                                   | security_ips_src                                                         
+--------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------
| ["127.0.0.1","100.104.106.0/24,100.104.160.192/26,100.104.188.0/24,11.59.10.2,11.59.10.3,11.59.6.125,11.59.6.126","0.0.0.0/1"] | [{"DBInstanceIPArrayAttribute":"","DBInstanceIPArrayName":"default","Secu
+--------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------
> ```

**Comment by cbruno10** [comment]
@bigdatasourav In the example output above, can you please edit to include a non-truncated version?

Also, can you please replace the example you have in the doc with a query for:
`### List DB instances that allow 0.0.0.0/0 `

**Review by cbruno10 — CHANGES_REQUESTED** [review]
Please see PR comment, thanks!

**Comment by bigdatasourav** [comment]
```
> select db_instance_id, security_ips from alicloud_rds_instance where security_ips :: jsonb ? '0.0.0.0/0'
+----------------------+--------------------------------------------------------------------------------------------------------------------------------+
| db_instance_id       | security_ips                                                                                                                   |
+----------------------+--------------------------------------------------------------------------------------------------------------------------------+
| rm-7goaou30dq55i7p65 | ["127.0.0.1","100.104.106.0/24,100.104.160.192/26,100.104.188.0/24,11.59.10.2,11.59.10.3,11.59.6.125,11.59.6.126","0.0.0.0/0"] |
+----------------------+--------------------------------------------------------------------------------------------------------------------------------+
>

**Review by cbruno10 — APPROVED** [review]

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**